tests/Integration/Subscriber/Addons/CloudflareSubscriber/TestSetVarnishLocalhost.php

```diff
@@ -20,11 +27,14 @@ public function testShouldReturnDefaultWhenCloudflareDisabled() {
 		$cf_subscriber     = new CloudflareSubscriber( new Cloudflare( new Options_Data( (new Options( 'wp_rocket_'))->get( 'settings' ) ), $cloudflare_facade ), new Options_Data( (new Options( 'wp_rocket_'))->get( 'settings' ) ), new Options() );
 
 		$this->assertSame(
-			'',
-			$cf_subscriber->set_varnish_localhost( '' )
+			[],
+			$cf_subscriber->set_varnish_localhost( [] )
```

Comment:
@Tabrisrp Instead of directly invoking the method, find a way to test in an integrated fashion.